test/test-server/cmk_handlers.go

```diff
@@ -42,6 +42,9 @@ func handleCmkKafkaClusterCreate(t *testing.T) http.HandlerFunc {
 			if *req.Spec.DisplayName == "gcp-byok-test" {
 				cluster.Spec.Config.CmkV2Dedicated.EncryptionKey = cmkv2.PtrString("xyz")
 			}
+			if *req.Spec.DisplayName == "cck-byok-test" {
```

Comment:
```suggestion
			if req.Spec.GetDisplayName() == "cck-byok-test" {
```

Reply:
I think I just aligned myself to how you did it in your PR there just two lines above. Fine with the change, took the freedom to change that for the code above as well.

Reply:
Thanks so much! It's not a huge issue since this is in our tests, but `nil` pointer errors are something we've had a few problems with recently.